src/sentry/integrations/slack/utils/notifications.py

```diff
@@ -63,12 +69,60 @@ def send_incident_alert_notification(
     }
 
     client = SlackClient(integration_id=integration.id)
+    parent_notification_message = None
     try:
-        client.post("/chat.postMessage", data=payload, timeout=5)
-        return True
-    except ApiError:
+        parent_notification_message = repository.get_parent_notification_message(
+            alert_rule_id=incident.alert_rule_id,
+            incident_id=incident.id,
+            trigger_action_id=action.id,
+        )
+    except Exception:
+        # if there's an error trying to grab a parent notification, don't let that error block this flow
+        pass
```

Comment:
Replying to threads is a nice-to-have, not a need-to-have, so we shouldn't block the notification from sending, which is why we don't block in this area